app/services/storage.js

```diff
@@ -18,28 +18,49 @@
  * @module Storage
  */
 
-import * as sqlite from './sqlite';
+import cbmkdirp from 'less-mkdirp';
+import path from 'path';
+import thenify from 'thenify';
 
-/**
- * Open a database and return a promise-based wrapper.
- * @param path
- * @returns {Promise}
- */
-export function open(path) {
-  sqlite.verbose();
-
-  const o = sqlite.open(path);
-
-  // As a side-effect, log the user version.
-  o.then((db) => {
-    db.get('PRAGMA user_version')
-      .then((row) => {
-        console.log(`User version: ${row.user_version}.`);
-      })
-      .catch((err) => {
-        console.log(`Unable to fetch DB user_version: ${err}.`);
-      });
-  });
-
-  return o;
+import { DB, verbose } from './sqlite';
+
+const mkdirp = thenify(cbmkdirp);
+
+export class Storage {
```

Comment:
Class comment -- how is `Storage` different from ... Why is it not `ProfileStorage` or `EventStore` or ...